A32NX/ModelBehaviorDefs/A32NX/Interior/A32NX_Interior_Misc.xml

```diff
@@ -109,7 +109,7 @@
 
     Takes into account the value of L:XMLVAR_SWITCH_OVHD_INTLT_ANNLT_Position.
 
-    Authors: Asobo / Markus Ullmann (@jokey2k) 18/10/20
+    Authors: Asobo / Markus Ullmann (@jokey2k) 18/10/20 Geoff Darst(geoffda) 2/11/20   
```

Comment:
I could be wrong, but use the `changelog.md` instead

Reply:
Done.